crabbitt42/git/aboutme.txt

```diff
@@ -0,0 +1,3 @@
+Hi!
+
+My name's Chris. I'll be missing the first part of next week's lesson. Anything I should know?
```

Comment:
Hmm, not sure. We'll have the stuff online so you can always follow that later. And we'll be using a lot of the concepts throughout the workshops, so you'll get practice. You could always come 20 min earlier to next weeks session to catch up.